src/upload/src/Upload.tsx

```diff
@@ -312,22 +314,35 @@ export default defineComponent({
     }
     function handleFileAddition (files: FileList | null, e?: Event): void {
       if (!files) return
-      Array.from(files).forEach((file) => {
-        const fileInfo: FileInfo = {
-          id: createId(),
-          name: file.name,
-          status: 'pending',
-          percentage: 0,
-          file: file,
-          url: null
-        }
-        doChange(fileInfo, e, {
-          append: true
+      const file = Array.from(files)
+      const {onBeforeUpload} = props
```

Comment:
```suggestion
      const { onBeforeUpload } = props
```

格式不对，可能你的 git hook 需要重新配置一下，husky

Reply:
好的，明天会设置一下。